packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -78,9 +80,30 @@ public void updateBuilder(@NonNull CaptureRequest.Builder requestBuilder) {
     if (!checkIsSupported()) {
       return;
     }
-    requestBuilder.set(
-        CaptureRequest.CONTROL_AE_REGIONS,
-        exposureRectangle == null ? null : new MeteringRectangle[] {exposureRectangle});
+
+    try {
+      if (exposureRectangle != null) {
+        requestBuilder.set(
+            CaptureRequest.CONTROL_AE_REGIONS, new MeteringRectangle[] {exposureRectangle});
+      } else if (defaultRegions != null && defaultRegions.length > 0) {
+        requestBuilder.set(CaptureRequest.CONTROL_AE_REGIONS, defaultRegions);
+      } else if (!defaultRegionsHasBeenSet) {
+        setDefaultRectangleAndSetBuilder(requestBuilder);
```

Comment:
You should be able to get the default value by using the `get` method:  `requestBuilder.get(CaptureRequest.CONTROL_AE_REGIONS)`. And this should be called before the try/catch:

```java
if (!defaultRegionsHasBeenSet) {
  defaultRegions = requestBuilder.get(CaptureRequest.CONTROL_AE_REGIONS);
  defaultRegionsHasBeenSet = true;
}
```

Reply:
i forgot add this lines

Reply:
Since you are setting the default value above now, you only need to check whether the `exposureRectangle` is null.

```java
if (exposureRectangle != null) {
  requestBuilder.set(
      CaptureRequest.CONTROL_AE_REGIONS, new MeteringRectangle[] {exposureRectangle});
} else {
  requestBuilder.set(CaptureRequest.CONTROL_AE_REGIONS, defaultRegions);
} 
```

And you shouldn't need to wrap it in a try/catch.